packages/action-menu/test/action-menu-groups.test.ts

```diff
@@ -91,12 +90,7 @@ describe('Action Menu - Groups', () => {
             press: 'ArrowDown',
         });
         await opened;
-        expect(el.open).to.be.true;
```

Comment:
When opened, menu should focus on the first selected item, so we don't need to `ArrowUp` to focus on it

Reply:
removed so many of these in the menu test